data/uk.co.mrbenshef.Boop-GTK.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Boop-GTK
```

Comment:
This should be part of the source tree, it's not a flatpak/flathub specific thing.